src/wasm.h

```diff
@@ -59,6 +59,7 @@
 #include "pretty_printing.h"
 #include "support/bits.h"
 #include "support/utilities.h"
+#include "support/metrics.h"
```

Comment:
I'd like to not add things in `wasm.h` - for example, we moved printing out. Do we need metrics here?